src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -238,6 +321,33 @@ You can query information about a specific stream rule by querying for the accou
 }
 ```
 
+Azure Event Hub:
+```graphql
+{
+  actor {
+    account(id: YOUR_NR_ACCOUNT_ID) {
+      streamingExport {
+        streamingRule(id: "INSERT_RULE_ID") {
+          azure {
+            eventHubConnectionString
```

Comment:
Should we mention this will be returned mostly masked?

Reply:
I was considering mentioning that, but I think that the mask is an intuitive enough user experience? We still return the last 5 characters 🤔